### ISSUE TITLE
Pressing tab in a snippet inserts a tab instead of moving to the next placeholder

### ISSUE THREAD

**Issue by hesselmonk** [microsoft/vscode-docs-authoring#564]
- VSCode Version: 1.44.2
- OS Version: Windows_NT x64 10.0.18362

Steps to Reproduce:

1. Add a user snippet for markdown, e.g.
```
	"Snippet": {
		"prefix": "snpt",
		"body": [
			"# _[${0}](${1})_"
		]
	},
```
2. In a markdown file, enter the snippet and (optionally insert text) press tab. Instead of moving the cursor to the next placeholder, a tab is inserted.

**Comment by adunndevster** [comment]
@hesselmonk, how are you adding a "user snippet" via docs-markdown? Just entering that raw text?

**Comment by cangulo** [comment]
I am facing the same issue, this is a screenshot of the snippet I'm using

![image](https://user-images.githubusercontent.com/23151731/81496107-63d71f80-92b5-11ea-8c34-0fff4af54d06.png)

**Comment by bharney0** [comment]
@adunndevster @jamarw looks like this has been reported before in issue #140. We should look into this.

**Comment by bharney0** [comment]
AB#227365

**Comment by jamarw** [comment]
Hi @hesselmonk , this issue was fixed in release 0.2.73. Thanks!

![snippet-tab](https://user-images.githubusercontent.com/11825888/84523591-639fba80-ac8d-11ea-94f9-8dcb9426d3e2.gif)

**Comment by nanachi-code** [comment]
i got this problem again today, can someone look into it?
